File: Java_Python_Programming/starters/problem3/compare_two_numbers.py
# Compare two numbers and print the larger one or indicate if they are equal.
# def compareTwoNumbers(a, b):
#     if a > b:
#         print(f"{a} is greater than {b}")
#     elif b > a:
#         print(f"{b} is greater than {a}")
#     else:
#         print("Both numbers are equal")
# num1 = int(input("Enter the first number: "))
# num2 = int(input("Enter the second number: "))
# compareTwoNumbers(num1, num2)

def compareGivenNumbers(a, b):
    if a == b:
        return "Both numbers are equal"
    return f"{a} is greater than {b}" if a > b else f"{b} is greater than {a}"
num1 = int(input("Enter the first number: "))
num2 = int(input("Enter the second number: "))
print(compareGivenNumbers(num1, num2))

File: Java_Python_Programming/starters/problem3/compare_two_numbers.java
package starters.problem3;

import java.util.Scanner;

// Compare two numbers and print the larger one or indicate if they are equal.
public class compare_two_numbers {
    public static Scanner scanner = new Scanner(System.in);
    public static void main(String[] args) {
        System.out.println("Enter the first number: ");
        int num1 = scanner.nextInt();
        System.out.println("Enter the second number: ");
        int num2 = scanner.nextInt();
        if (num1 == num2) {
            System.out.println("Both numbers are equal.");
            return;
        }
        int largerNum = num1 > num2 ? num1 : num2;
        System.out.println("The larger number is: " + largerNum);
    }
}
